.circleci/config.yml

```diff
@@ -155,7 +155,7 @@ jobs:
             dpkg -l > /logs/versions.txt
             conda env export -n base > /logs/build_environment.yml
             # Build conda package
-            conda build package -c conda-forge -c esmvalgroup > /logs/build_log.txt
+            conda build package -c conda-forge -c conda-forge/label/rc_iris -c esmvalgroup > /logs/build_log.txt
```

Comment:
note that this is not needed outside the scope of testing (which is fine, since we tested with Github Actions); I would take it out together with all the references to the `label` channel after you are happy all tests pass on Circle since we'll merge the iris3 specific changes only when they have released it fully on conda-forge :beer:

Reply:
Will do, but since we expect some more changes, especially once we merge master before merging, it would be useful if the CircleCI tests give the output here.

Reply:
yup, good call!

Reply:
I'm also trying to learn how this stuff works, so it's good practice 😁👍